x-pack/plugin/esql/src/main/java/org/elasticsearch/xpack/esql/plan/logical/Enrich.java

```diff
@@ -145,7 +146,7 @@ public void writeTo(StreamOutput out) throws IOException {
         out.writeNamedWriteable(policyName());
         out.writeNamedWriteable(matchField());
         if (out.getTransportVersion().before(TransportVersions.V_8_13_0)) {
-            out.writeString(BytesRefs.toString(policyName().fold())); // old policy name
+            out.writeString(BytesRefs.toString(policyName().fold(FoldContext.unbounded() /* TODO remove me */))); // old policy name
```

Comment:
Going by the grammar, I don't think the enrich policy name can be an actual expression in the first place. In any case, folding this before serializing the plan sounds quite bad :/

Reply:
Right. This is only to 8.12.x so maybe it's ok.

Reply:
That should be a nice little work item for some later follow-up.